pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "assetman"
+description = "assetman assetmanager"
+version = "0.3.0rc"
+authors = [
+  { name="Will McCutchen", email="wm@bit.ly" },
+]
+maintainers = [
+  { name="Jehiah Czebotar", email="jehiah@gmail.com"},
+  { name="Josh Harshman", email="josh.harshman@bit.ly"}
+]
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+  "aadict==0.2.3",
+  "asgiref==3.7.2",
+  "asset==0.6.13",
+  "boto3==1.34.15",
+  "botocore==1.34.15",
+  "exceptiongroup==1.2.0",
+  "globre==0.1.5",
+  "iniconfig==2.0.0",
+  "jmespath==1.0.1",
+  "packaging==23.2",
+  "pluggy==1.3.0",
+  "pytest==7.4.4",
+  "python-dateutil==2.8.2",
+  "s3transfer==0.10.0",
+  "six==1.16.0",
+  "sqlparse==0.4.4",
+  "tomli==2.0.1",
+  "tornado==6.3.3",
+  "typing-extensions==4.9.0",
+  "urllib3==1.26.18",
```

Comment:
I expect just `tornado` and `boto3` here - those are the only 3rd party packages we import. 

I feel like this should list min versions we expect to be compatible with using `>=` to cause the least problems for consumers of this pacakge.

Reply:
I switched it to use `>=`. Not sure about what min versions to use though